projects/ltc7841/Makefile

```diff
@@ -0,0 +1,9 @@
+# Select the example you want to enable by choosing y for enabling and n for disabling
+BASIC_EXAMPLE = n
+IIO_EXAMPLE = y
```

Comment:
You need to remove this and replace with maybe a default like this:
EXAMPLE ?= IIO_EXAMPLE